src/finch/tensor.py

```diff
@@ -515,6 +543,39 @@ def asarray(obj, /, *, dtype=None, format=None):
         return tensor
 
 
+def full(
+    shape: int | tuple[int, ...],
+    fill_value: jl_dtypes.number,
+    *,
+    dtype: DType | None = None,
+) -> Tensor:
+    if isinstance(shape, int):
+        shape = (shape,)
+    if dtype is not None:
+        fill_value = dtype(fill_value)
+    return Tensor(jl.Tensor(jl.SparseCOO[len(shape)](jl.Element(fill_value), shape)))
```

Comment:
pass in shape as an argument to `jl.Tensor` to initialize it.

Reply:
Done!